src/modules/users/users.module.ts

```diff
@@ -0,0 +1,19 @@
+import { forwardRef, Module } from '@nestjs/common';
+import { UserController } from './users.controller';
+import { UsersService } from './users.service';
+import { UsersRepository } from './users.repository';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { customError } from 'messages/errors';
+import { AuthModule } from '../auth/auth.module';
+
+@Module({
+  imports: [
+    TypeOrmModule.forFeature([UsersRepository]),
+    customError,
```

Comment:
custom error should't be here, recheck it

- [ ] remove custom error from module import